tabbycat/results/templates/public_ballot_set.html

```diff
@@ -72,29 +72,11 @@ <h4 class="card-title">
           {% for team in sheet.teams %}
             <div class="col-6 list-group mb-3">
 
-              {% for position in team.speakers %}
-                <li class="list-group-item">
-                  <strong>{{ position.name }}</strong>
-                  {{ position.speaker }}
-                  <span class="badge badge-secondary float-right">
-                    {{ position.score }}
-                  </span>
-                </li>
-              {% endfor %}
-              <li class="list-group-item list-group-item-{{ team.win_style }}">
-                {% if use_code_names %}
-                  {% blocktrans trimmed with name=team.team.code_name side=team.side %}
-                    <em>Total for {{ name }} ({{ side }})</em>
-                  {% endblocktrans %}
-                {% else %}
-                  {% blocktrans trimmed with name=team.team.short_name side=team.side %}
-                    <em>Total for {{ name }} ({{ side }})</em>
-                  {% endblocktrans %}
-                {% endif %}
-                <span class="badge badge-secondary float-right">
-                  {{ team.total }}
-                </span>
-              </li>
+              {% if pref.elim_ballot_nom and debate.round.is_break_round %}
```

Comment:
ditto re what happens if `pref.teams_in_debate == 'bp' and not pref.elim_ballot_nom`.